examples/pulse_control/6_Use_jobs_with_pulse_sequence_sweeps.ipynb

```diff
@@ -0,0 +1,409 @@
+{
```

Comment:
<p>1) Lets make sure that we say Hybrid Job or Hybrid Jobs in place of just job. People find "job" confusing by itself so we have to make it better for them </p><p> </p><p>2) This is one of the first <strong>steps</strong> to...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-braket-examples/pull/294/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>